events/ME_Australia_Events.txt

```diff
@@ -98,7 +98,6 @@ country_event = { # §[Root.GetAdjective]§! Rise to Power
 	desc = ME_Australia_Events.2.D
 	picture = ENLIGHTENMENT_eventPicture
 
-	is_triggered_only = yes
```

Comment:
Searched for `ME_Australia_Events.2` and did not find an event call for it in the mod files.

![image](https://user-images.githubusercontent.com/5704491/91648552-c723c380-ea04-11ea-810e-81565f5bb85e.png)